phonebuddies/EmailDraft.py

```diff
@@ -63,3 +63,14 @@ def draft_overdue_process_admin_reminder(email):
 """
 
         return EmailDraft(email, "ACTION NEEDED: AESOP Phone Buddy Reminder", message)
+
+    def draft_admins_without_buddy_emails(email):
+            message = f"""This is a friendly reminder that you are an administrator without a buddy. The dry-run of buddy emails has been sent, and 
```

Comment:
Remove language around dry-run—this will only be sent when `really_send_emails` is true anyways.